### DISCUSSION TITLE
Formula columns disappeared after fetching tables

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#8529] [Help] [answered]
Hi,
I am using a Postgresql external db. We have a new table added and after clicking on the "Fetch Tables" button to refresh the tables. However I discovered this operation has also wiped out all the formula columns.

Is there a way to recover this? The only option I see is to re-import the previous export and go through each table to find all the formula columns again and re-enter them.

**Comment by melohagan** [comment, marked as answer]
Hey @micebrain 

Sorry to hear that happened, were there many Formulas that got wiped?

A feature was added to somewhat get around this in that you can select specifically which tables you want to fetch:
![Screenshot 2022-11-07 at 11 53 22](https://user-images.githubusercontent.com/101575380/200303787-3849e588-5b17-448f-a756-9f793750f1eb.png)

This will still wipe the formulas from the fetched tables, but it means if you had other tables with formulas then those tables will be left untouched.

**Reply by micebrain** [reply]
Oh, I missed this completely :( 
I have about 10 formulas, so it's not terribly bad. But will keep this in mind for the future.
Thanks!
